src/frontend/components/home/CategoryCard.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+const CategoryCard = ({ category }) => {
+  return (
+    <div className="categoryCard bg-slate-900">
+      <img src={category.categoryImg} alt={category.categoryName} />
+      <video autoPlay={true} loop={true} playsInline={true}>
+        <source src={category.categoryVideo} type="video/mp4" />
```

Comment:
Suggestion : You can destructure category instead of repeating it again and again